### ISSUE TITLE
I can not play games on my 4K laptop because my eyes are sick.

### ISSUE THREAD

**Issue by iyagicom** [OpenRA/OpenRA#16733]
#### System Information
- **Operating System:** Ubuntu 19.04
- **.NET / Mono Version:
- **OpenRA Version:20190314
- **Mod: Red Alert, Tiberian Dawn, Dune2000

The game screen is not important.
The interface is too small and my eyes are sore.
Let me double the size in the graphics options.

**Comment by pchote** [comment]
You can define an environment variable `OPENRA_DISPLAY_SCALE=2` (either in your `.bash_profile` or directly when running the AppImage) to enable HiDPI mode. Enabling this as a general option depends on #10382.

**Comment by iyagicom** [comment]
OPENRA_DISPLAY_SCALE = 2 in .profile does not work.
Add 2X options in the settings display.

**Comment by abcdefg30** [comment]
> Add 2X options in the settings display.

Zoom will be available once #10382 is fixed, so closing in favour of that ticket.
